src/navigation/services/buy-crypto/screens/BuyCryptoRoot.tsx

```diff
@@ -369,40 +377,48 @@ const BuyCryptoRoot: React.FC<
     let title, message: string;
     switch (type) {
       case 'walletNotSupported':
-        title = 'Wallet not supported';
-        message =
-          'The selected wallet is currently not supported for buying cryptocurrencies';
+        title = t('Wallet not supported');
+        message = t(
+          'The selected wallet is currently not supported for buying cryptocurrencies',
+        );
         break;
       case 'needsBackup':
-        title = 'Needs backup';
-        message =
-          'The key of the selected wallet needs backup before being able to receive funds';
+        title = t('Needs backup');
+        message = t(
+          'The key of the selected wallet needs backup before being able to receive funds',
+        );
         break;
       case 'walletNotCompleted':
-        title = 'Incomplete Wallet';
-        message =
-          'The selected wallet needs to be complete before being able to receive funds';
+        title = t('Incomplete Wallet');
+        message = t(
+          'The selected wallet needs to be complete before being able to receive funds',
+        );
         break;
       case 'noWalletsAbleToBuy':
-        title = 'No wallets';
+        title = t('No wallets');
         message = coin
-          ? `No ${coin.toUpperCase()} wallets available to receive funds.`
-          : 'No wallets available to receive funds.';
+          ? t('No wallets available to receive funds.', {
```

Comment:
t('noAvailableWallets', {coin: coin.toUpperCase()})` and in translation.json `noAvailableWallets: No {{coin}} wallets available to receive funds.`